src/scrapers/CourseScraper.js

```diff
@@ -0,0 +1,90 @@
+import * as cheerio from'cheerio';
+
```

Comment:
- Change the file name to be "lectureParser.ts"
- leave the folder "scrapers" as it is, I will later on change it to "parsers"